### PR TITLE
Replace instructions with links, remove nitrous section, remove check…

### PR DESCRIPTION
**Pull request by jhack32**
…out ... line, remove postgres

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,23 +20,11 @@ This project is emphatically beginner-friendly so, if you're interested in poten
...
 This site runs on **Ruby 2.0.0-p353** and **Rails 4.0.0** (upgraded from Ruby 1.9.3 and Rails 3.2.12 in February 2014)
 
-#### Nitrous.io Setup
-
-We suggest that Windows users who can't migrate to Linux and newbies who are having trouble setting up their environments just use [nitrous.io](https://www.nitrous.io/join/GRrt3VYaHE8?utm_source=nitrous.io&utm_medium=copypaste&utm_campaign=referral) or a similar cloud-based environment like [Cloud 9](https://c9.io/). 
-
-For Nitrous:
-1. [Follow along with this gist](https://gist.github.com/afshinator/8035821) to do the setup on Nitrous.
-2. Don't forget to populate your test database with `$ rake db:test:prepare`
+#### Getting started
```
Comment (KevinMulhern) [inline README.md]: Could you change this title to "Installing the Site on your Local Machine"

Comment (jhack32) [inline README.md]: :+1:

File: README.md [hunk 2]
```diff
@@ -22,23 +20,11 @@ This project is emphatically beginner-friendly so, if you're interested in poten
...
 This site runs on **Ruby 2.0.0-p353** and **Rails 4.0.0** (upgraded from Ruby 1.9.3 and Rails 3.2.12 in February 2014)
 
-#### Nitrous.io Setup
-
-We suggest that Windows users who can't migrate to Linux and newbies who are having trouble setting up their environments just use [nitrous.io](https://www.nitrous.io/join/GRrt3VYaHE8?utm_source=nitrous.io&utm_medium=copypaste&utm_campaign=referral) or a similar cloud-based environment like [Cloud 9](https://c9.io/). 
-
-For Nitrous:
-1. [Follow along with this gist](https://gist.github.com/afshinator/8035821) to do the setup on Nitrous.
-2. Don't forget to populate your test database with `$ rake db:test:prepare`
+#### Installing the Site on your Local Machine
```
Comment (KevinMulhern) [inline README.md]: Also could you add some instructions below the title, something like
1. Make sure to fork this repo
2. When you have it cloned to your local machine make sure your Ruby version is 2.0.0-p353 and your Rails version is 4.0.0

Comment (KevinMulhern) [inline README.md]: and one final instruction under those 2,
3. Use the appropriate operating system focused guide below to install the site

**Comment by KevinMulhern** [comment]
@eriktrautman here is an updated README done by @jhack32 to include those guides for installing we did last year. Let us know what you think.

**Comment by eriktrautman** [comment]
Looks much more straightforward to me!

**Comment by KevinMulhern** [comment]
Great :+1: there are a few small adjustments that we need to do to the guides as described in this ticket https://trello.com/c/amDQiPhN/22-move-main-site-set-up-guides-over-to-be-odin-as-readme-s

**Comment by KevinMulhern** [comment]
@eriktrautman this one too please :)

**Pull request merged**